plane/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "plane"
 version = "0.2.0"
 edition = "2021"
+autotests = false
```

Comment:
why was this necessary?

Reply:
otherwise it looks for tests in /tests, but actually since we aren't providing another target...

Reply:
ah that's right it's cause I was calling plane-tests tests earlier and it would complain

Reply:
thanks

Reply:
but we no longer have a `/tests` directory at that level, right?